packages/rust-bindings/scripts/build-scripts.ts

```diff
@@ -41,3 +42,4 @@ export const convertWasmToJs = async (outPath: string) => {
             fs.writeFileSync(file, content, 'utf-8');
         });
 };
+*/
```

Comment:
What's the intention here? This is a required part of the compilation for react-native compatibility